packages/eas-cli/src/utils/prompts.ts

```diff
@@ -0,0 +1,90 @@
+import chalk from 'chalk';
+
+import { EnvironmentVariableEnvironment } from '../graphql/generated';
+import { promptAsync, selectAsync } from '../prompts';
+
+export async function promptVariableEnvironmentAsync(
+  nonInteractive: boolean
+): Promise<EnvironmentVariableEnvironment> {
+  if (nonInteractive) {
+    throw new Error('Environment may not be empty.');
+  }
+  return await selectAsync('Select environment:', [
+    { title: 'Development', value: EnvironmentVariableEnvironment.Development },
+    { title: 'Preview', value: EnvironmentVariableEnvironment.Preview },
+    { title: 'Production', value: EnvironmentVariableEnvironment.Production },
+  ]);
```

Comment:
```suggestion
  return await selectAsync('Select environment:', Object.keys(EnvironmentVariableEnvironment).map(key => ({
        title: key,
        value: EnvironmentVariableEnvironment[key]
    }));
```

Reply:
Ditto, IMO current version is clearer, you know what it does at first sight. Meanwhile, the approach with extracting keys and values from EnvironmentVariableEnvironment requires some decoding.